client/signup/main.jsx

```diff
@@ -762,6 +768,7 @@ class Signup extends Component {
 								flowName: this.props.flowName,
 								stepName: this.props.stepName,
 							} }
+							pageTitle={ this.getPageTitle() }
```

Comment:
Good catch! thanks!

Reply:
Do we want to show the page title in every flow, if no back buttons are present? 
If so I think we can use `pageTitle={ this.props.pageTitle }`, otherwise it'd be good to maintain the check that @renancarvalho added :)

Reply:
I think it is the other way around, we just show the page title for tailored flow( LiB and NL ), the 'normal' is the back button which has two 'versions' one 'back to sites' and the other one just 'back'. ( just kept the original behaviour )

@alshakero how do you think it should be?